src/converter.js

```diff
@@ -5,6 +5,9 @@ var ass = require('./ass');
 exports.convert = function (_in, _out, _options, callback)
```

Comment:
We need to make sure it's clear that argument is optional

so I'd either do this:
`exports.convert = function (_in, _out, _options, callback = undefined)`

Or move the callback
```
function defaultCallback(error) {
  if (error) throw error;
}

exports.convert = function (_in, _out, _options, callback = defaultCallback)
```

Reply:
I kept the second option since the first one did not solve the problem (undefined is still not a function).